Xamarin.Forms.Core/Registrar.cs

```diff
@@ -194,7 +194,16 @@ public static void RegisterAll(Type[] attrTypes)
 			{
 				foreach (Type attrType in attrTypes)
 				{
-					Attribute[] attributes = assembly.GetCustomAttributes(attrType).ToArray();
+					Attribute[] attributes;
+					try
+					{
+						attributes = assembly.GetCustomAttributes(attrType).ToArray();
+					}
+					catch (System.IO.FileNotFoundException)
+					{
+						// Sometimes the previewer doesn't actually have everything required for these loads to work
+						continue;
```

Comment:
Same - need to log this. Or, better yet, have some sort of indication we're running under the Previewer so we know this is an exception we can ignore.